src/Agent/NewRelic/Agent/Core/OpenTelemetryBridge/MeterListenerBridge.cs

```diff
@@ -165,10 +183,41 @@ public void Start()
                     // Configure metric collection and export settings
                     metricReaderOptions.PeriodicExportingMetricReaderOptions.ExportIntervalMilliseconds = DefaultOtlpExportIntervalSeconds * 1000;
                     metricReaderOptions.TemporalityPreference = MetricReaderTemporalityPreference.Delta;
-
                 });
 
-                _meterProvider = providerBuilder.Build();
+            return providerBuilder.Build();
+        }
+
+        /// <summary>
+        /// Starts the meter listener bridge to begin collecting and forwarding metrics.
+        /// Initializes the OpenTelemetry SDK, configures OTLP export, and sets up the MeterListener.
+        /// </summary>
+        public void Start()
+        {
+            if (!_configuration.OpenTelemetryMetricsEnabled)
```

Comment:
Do we also need to check the global OTel bridge enabled/disabled config setting here as well as the specific setting for metrics enabled/disabled?